docker-compose.yml~

```diff
@@ -0,0 +1,142 @@
+version: '3.3'
+services:
+    certs:
+        container_name: oe-certs 
+        image: itechuw/certgen:main
+        restart: always
+        environment:
+            - KEYSTORE_PW="kspass"
+            - TRUSTSTORE_PW="tspass"
+        networks:
+            - default
+        volumes:
+            -  key_trust-store-volume:/etc/openelis-global
+            -  keys-vol:/etc/ssl/private/
```

Comment:
Clean up this PR and only include what is required. i would advise to only focus on tickets  created by @mozzy11  to get more hands on skills.

Reply:
Am not understanding its my first time to push but every thing worked on my machine here is the tickethttps://github.com/I-TECH-UW/OpenELIS-Global-2/issues/708

Reply:
![Screenshot from 2024-04-25 21-49-06](https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/150329678/ec9f05e0-ce06-4a1d-b67f-d65f0d6fc371)
[Screencast from 25-04-24 21:42:23.webm](https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/150329678/f00a9f62-9cc0-4c38-9af3-fd9b3924b66f)